tracer/build/crank/README.md

```diff
@@ -0,0 +1,36 @@
+Crank
+=====
+
+The crank jobs are primarily designed to run as part of the CI, but it can be useful to run them from your local machine
+if you're trying to optimize something.
+
+The scripts `run.sh` and `run-appsec.sh` are design to be used at part of the CI.
+
+The script `./run-adhoc.sh` should be run from a Linux dd-trace-dotnot development environment (WSL will do nicely). It can be used
```

Comment:
```suggestion
The script `./run-adhoc.sh` should be run from a Linux dd-trace-dotnet development environment (WSL will do nicely). It can be used
```

Reply:
Though I'd love to see a Buzzfeed-style article called "DotNot: 10 Things Not To Do In Your .NET Applications. Number 5 will surprise you!"

Reply:
Could catch, thanks @zacharycmontoya